### ISSUE TITLE
numerical instability?

### ISSUE THREAD

**Issue by kjslag** [mgmalek/efficient_cross_entropy#7]
This [PR](https://github.com/mgmalek/efficient_cross_entropy/pull/5) adds a test where the logits are 10x the normal distribution. The test fails, which suggests a possible numerical instability.
